### PR TITLE
Resolve issue 555 and 558

### PR DESCRIPTION
**Pull request by matsduf**
* Updated the upgrade instructions in README to handle multiple upgrades (issue #558)
* Updated the upgrade instructions file under `docs/` (issue #558)
* Added two upgrade instructions files under `docs/` for two upgrades (issue #558)
* Added the latest upgrade files (issue #555) to MANIFEST

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,22 @@ instructions](https://github.com/zonemaster/zonemaster-engine/blob/master/docs/I
...
+follow the installation instructions. In some cases you need to patch the
+database when you update Zonemaster-Backend.
 
-For all other upgrades follow the relevant parts of the installation
-instructions below.
+Always take a backup if the database is valuable.
 
+Current version                     | Upgrade to version | Link to instructions
+------------------------------------|--------------------|--------------------------------------
+Older than 1.0.3                    | 1.0.3 or newer     | [upgrade-to-1.0.3]
```
Comment (mattias-p) [inline README.md]: It's not spelled out that you need to follow all three upgrade instructions if you're upgrading from `<1.0.3` to `>=5.0.0`.

File: README.md [hunk 2]
```diff
@@ -27,13 +27,22 @@ instructions](https://github.com/zonemaster/zonemaster-engine/blob/master/docs/I
...
+At least 1.0.3 but older than 1.1.0 | 1.1.0 or newer     | [upgrade-from-1.0.x-to-1.1.x]
+At least 1.1.0 but older than 5.0.0 | 5.5.0 or newer     | [upgrade-from-4.0.x-to-5.0.x]
```
Comment (mattias-p) [inline README.md]: 5.5.0 -> 5.0.0

File: README.md (hunk 1 above)
Comment (matsduf) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (matsduf) [reply README.md]: Fixed.

**Comment by matsduf** [comment]
@sandoche2k, please review.

**Comment by matsduf** [comment]
@sandoche2k, please review and I'll merge if you approve.

**Comment by sandoche2k** [comment]
@matsduf with three approvals, can you merge?

**Pull request merged**